### PR TITLE
CHK-316: Initial Setup

### PR DESCRIPTION
**Pull request by wdsrocha**
#### Description

The setup is based on [this document](https://docs.google.com/document/d/11DewV9j4qiz_U_tuLsAHUa235oydbcSupIkuVdMi9kg), https://github.com/vtex-apps/graphql-interface-app-example and https://github.com/vtex-apps/search-graphql (which is also an interface).

The schema is based on a subset of https://github.com/vtex-apps/places-graphql/ schemas (see https://github.com/vtex-apps/places-graphql/pull/5). After CHK-340 is complete, this subset of schemas on Places GraphQL will be removed.

#### How should this be manually tested?

Just check if the files are correct and make sense, there is not too much to test besides that. I suggest reviewing per commit.

### CODE REVIEW

File: graphql/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+type Query {
+  suggestAddresses(searchTerm: String): [AddressSuggestion]
+
+  getAddressByExternalId(id: String): Address
```
Comment (wdsrocha) [inline graphql/schema.graphql:4]: Is there a way to import the `Address` type from the https://www.github.com/vtex-apps/places-graphql ?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.vscode/
+.DS_Store
```
Comment (lucasecdb) [inline .gitignore:5]: isn't it missing more stuff? like node_modules etc

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+This app exports a GraphQL schema for geolocation results for Place Components.
+
+### Usage
+
+To use it in your app, declare it on your manifest file like:
+
+```
+"dependencies": {
+  "vtex.geolocation-graphql-interface": "0.x"
+}
```
Comment (lucasecdb) [inline docs/README.md]: this could be just
```suggestion
vtex add vtex.geolocation-graphql-interface
```

File: graphql/schema.graphql (hunk 1 above)
Comment (lucasecdb) [reply graphql/schema.graphql:4]: I don't think so :-/

File: .gitignore (hunk 2 above)
Comment (wdsrocha) [reply .gitignore:5]: I felt the same way, but as there is no `package.json`, there isn't much to add. On the example project there is an extensive `.gitgnore`, but looks exaggerated and unnecessary to me. What do you say?

the file: https://github.com/vtex-apps/graphql-interface-app-example/blob/c78a9a1a2a4f197dccfd302eb36088beb97a4eb7/graphql-interface-app/.gitignore

Comment (lucasecdb) [reply .gitignore:5]: I'd rather add it now because we might want something like prettier to auto format the graphql queries

**Review by lucasecdb — APPROVED** [review]

**Review by jeffersontuc — APPROVED** [review]

**Pull request merged**